modules/mongodb/manifests/s3backup/backup.pp

```diff
@@ -46,20 +46,22 @@
   $aws_secret_access_key = undef,
   $aws_region = 'eu-west-1',
   $backup_dir = '/var/lib/s3backup',
+  $backup_node = 'localhost',
   $cron = true,
   $env_dir = '/etc/mongo_s3backup',
   $private_gpg_key = undef,
   $private_gpg_key_fingerprint,
   $s3_bucket  = 'govuk-mongodb-backup-s3',
-  $user = 'govuk-backups'
+  $s3_bucket_daily = 'govuk-mongodb-backup-s3-daily',
+  $user = 'govuk-backup'
 ){
 
+  include backup::client
+
   validate_re($private_gpg_key_fingerprint, '^[[:alnum:]]{40}$', 'Must supply full GPG fingerprint')
 
-  # create user
-  user { $user:
-    ensure     => 'present',
-    managehome => true,
+  user { 'govuk-backups':
+    ensure => absent,
```

Comment:
If we can put a FIXME comment on top of this so we know to remove it when it's deployed to Production that'd be grand